### DISCUSSION TITLE
I'd like to request a file listing with data from within the respective file

### DISCUSSION THREAD

**Discussion by ghost** [gsantner/markor#1587] [Ideas]
Dear all,

I'd like to ask for an viewing option for the file listing. I am using Zettlr (see [here](https://docs.zettlr.com/en/) for it's docs) on the PC, where it is possible to list the files in the form of 3 lines, e.g.:

![image](https://user-images.githubusercontent.com/17848481/153365120-a7cf1d21-6cc0-4ae7-9061-113f3ff19b2d.png)

```
< first h1-Header of the document, if existing, else file name> < last edited>
< tags, only one line, cutoff if there are to many>
< real file name > < number of words >
```

at least the display of the header (as in the example `# Python snippets` ) would be very helpful. According to the Zettelkasten "rules", I am storing the files using the "Zettel"-ID as filename, which in the present folder view of Markor looks rather cryptic.

Of course, one could use filenames that give a hint to the content, but this quickly becomes unusable as the number of files grows. At least that's what "Zettelkasten"-experts recommend and I agree with that. A filename that makes perfect sense to me today, might become not helpful after some time. I do not want to spent time on finding the "best" filename, I just want to concentrate on my notes.
Therefore, I'd like to stay with the recommendation of using the "Zettel"-ID as filename.
For content relations and the ability to find something I am using the file view first, after that tag-filtering and, if this does not help, Search (moreover Zettlr suggests related files automagically, :-), but that's another issue).

Compared to other FOSS markdown editors, Markor appears to me most powerful. On the other hand, there is no app in the FOSS *Android* universe that supports the principles of the Zettelkasten-Method to a large extent. 

By addition of a more detailed file view, a suggested here, Markor could start to fill this gap.

Thanks a lot in advance
Markus

**Comment by gsantner** [comment]
@w-markus  Thanks for moving it and taking your time to write :-)

Copy from initial issue  #1585 

![grafik](https://user-images.githubusercontent.com/6735650/153503769-5192d84e-94d3-4041-b566-7aa45d1121d6.png)
